src/lib/components/primitives/button/button.story.svelte

```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
-  import Button from "./button.svelte";
-  import { initialize } from '$widget/styles.store';
+  /* eslint @typescript-eslint/no-empty-function: "off" */
```

Comment:
I'm conditionally turning off this ESLint rule within some `story` or `stories` files as some components require a "on click" or "on change" function that I don't need in our stories. I'd like the ability to just pass `() => {}` as the function.